app/components/appcomponents/MyAccount.js

```
@@ -161,27 +164,36 @@ const MyAccount = () => {
-            </svg>
-            {/* Text */}
-            <span className="relative cursor-pointer" onClick={() => setIsShowCards(true)}>
-              Moje darilne kartice
-              <span className="absolute -bottom-1 left-0 w-full h-[2px] bg-gradient-to-r from-blue-400 via-indigo-500 to-purple-600 rounded opacity-0 group-hover:opacity-100 transition-opacity duration-300"></span>
-            </span>
-          </span>
+          <>
+            {digiCards.map((digiCard) => {
+              return (
+                <span
+                  className="inline-flex items-center text-[14px] text-[#4B5563] font-variation-customOpt14 mobileUserAcc:text-[#9CA3AF] mobileUserAcc:text-[13px] mobileUserAcc:font-variation-customOpt13 font-semibold mb-1 space-x-3 transition-colors duration-300 hover:text-[#2563EB]">
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)